forgerock-auth/src/androidTest/java/org/forgerock/android/auth/webauthn/MockWebAuthnAuthenticationCallback.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2021 ForgeRock. All rights reserved.
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license. See the LICENSE file for details.
+ */
+
+package org.forgerock.android.auth.webauthn;
+
+import android.app.PendingIntent;
+
+import androidx.annotation.NonNull;
+
+import com.google.android.gms.fido.fido2.Fido2ApiClient;
+import com.google.android.gms.fido.fido2.api.common.PublicKeyCredentialRequestOptions;
+import com.google.android.gms.tasks.OnSuccessListener;
+import com.google.android.gms.tasks.Task;
+
+import org.forgerock.android.auth.callback.WebAuthnAuthenticationCallback;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+/**
+ * Extend the {@link WebAuthnAuthenticationCallback} and provide mocking result
+ * from {@link Fido2ApiClient}
+ */
+public class MockWebAuthnAuthenticationCallback extends WebAuthnAuthenticationCallback {
+
+    public MockWebAuthnAuthenticationCallback() {
+    }
+
+    public MockWebAuthnAuthenticationCallback(JSONObject jsonObject, int index) {
```

Comment:
Missing javadoc

Reply:
I would skip this javadoc for the testing class